src/host/os_getversion.c

```diff
@@ -75,10 +75,14 @@ SYSTEM_INFO getsysteminfo()
 
 void getversion(struct OsVersionInfo* info)
 {
-	OSVERSIONINFOEX versionInfo = {0};
+	RTL_OSVERSIONINFOEXW versionInfo = {0};
 
-	versionInfo.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
-	GetVersionEx((OSVERSIONINFO*)&versionInfo);
+	versionInfo.dwOSVersionInfoSize = sizeof(RTL_OSVERSIONINFOEXW);
+
+	typedef LONG (WINAPI* RtlGetVersionSig)(PRTL_OSVERSIONINFOEXW);
+	RtlGetVersionSig getVersion = (RtlGetVersionSig)
+	GetProcAddress(GetModuleHandle(TEXT("ntdll.dll")), "RtlGetVersion");
+	if ( getVersion ) getVersion(&versionInfo);
```

Comment:
According to [this](https://msdn.microsoft.com/en-us/library/mt723418(v=vs.85).aspx), you only need to include `ntddk.h` and link `ntdll.lib`. I personally don't see an issue with linking Windows libraries in, I prefer the code to be simple.